### DISCUSSION TITLE
Pimcore\Cache::save doesn't store content? Bug?

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#17172] [Q&A / Support]
If you look at the Pimcore docs concerning custom data caching (https://pimcore.com/docs/platform/Pimcore/Development_Tools_and_Details/Cache/) the documented code doesn't work for me:

`\Pimcore\Cache::save(
        $data,
        $cacheKey,
        ["output","tag1","tag2"],
        $lifetime);`

I have the following problems:

- If used like this the data is not stored in the cache, on reload the data is not in the cache
- As soon I add the priority option > 0 and pass an empty tags array it works, the data ist stored
- If I add the priority option > 0 and pass tags it doesn't work

Can anyone confirm this behaviour and this is a bug?

**Comment by akno77** [comment]
Nobody else having this issue? If it is a general bug this might affect the pimcore performance because in the core at some points there is also no priority used for caching so it might not be cached there, too!

**Comment by markus-moser** [comment]
Pimcore by default stores a maximum of 50 cache items within one process to avoid performance problems (see https://github.com/pimcore/pimcore/blob/11.x/lib/Cache/Core/CoreCacheHandler.php#L120).

Could it be the case that this limit gets reached in your constellation? If you have a very important cache entry which should not be limited by the limit but saved anyways you could set the `$force` parameter of the `\Pimcore\Cache::save()` method to true.
